packages/taboule/src/state/types.ts

```diff
@@ -16,3 +16,25 @@ export const TabouleQueryKey = t.union(
   'TabouleQueryKey'
 );
 export type TabouleQueryKey = t.TypeOf<typeof TabouleQueryKey>;
+
+export interface selectedRecommendation {
```

Comment:
This is actually already available as [ParsedInfo](https://github.com/tracking-exposed/trex/blob/master/platforms/yttrex/shared/src/models/Metadata.ts#L5)